jdaviz/configs/default/plugins/export/export.py

```diff
@@ -744,6 +747,31 @@ def save_movie(self, viewer, filename, filetype, i_start=None, i_end=None, fps=N
 
         return filename
 
+    def save_subset_as_stcs(self, filename):
+        """
+        Save a subset region to a text file in STC-S format.
+
+        Parameters
+        ----------
+        filename : str
+
+        Raises
+        ------
+        RuntimeError
+            If data is not aligned by WCS, which is required for STC-S export.
+        """
+        align_by = getattr(self.app, '_align_by', None)
+        if align_by != 'wcs':
+            raise RuntimeError("Please link datasets by WCS first using the Orientation plugin.")
+
+        region = self.app.get_subsets(subset_name=self.subset.selected,
+                                      include_sky_region=True)[0]['sky_region']
+
+        stcs_str = region2stcs_string(region)
+
+        with open(filename, 'w') as f:
+            f.write(stcs_str)

```

Comment:
@tomdonaldson – is there a convention for the file extension of STC-S strings written-to-disk?

Reply:
> @tomdonaldson – is there a convention for the file extension of STC-S strings written-to-disk?

@bmorris3 - No, there is no convention for storing them on their own.  In the wild the values usually show up in cells of a larger document in votable (.xml) or csv format.  

A single cell or value containing more than one shape implies a region that unions those shapes, so a single file defining one region would contain a single line with one or more space-separated stc-s shapes.  While it wouldn't help in recognizing the files by type, storing a single region in a csv file could have the benefit of being easy to work with and being recognized easily by some existing tools, and it would be extensible to adding metadata and multiple regions.